akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -309,6 +310,37 @@ abstract class UntypedPersistentActor extends UntypedActor with Eventsourced wit
 abstract class AbstractPersistentActor extends AbstractActor with PersistentActor with Eventsourced {
 
   /**
```

Comment:
PLS REVIEW

Note the naming of these methods, `defineReceiveRecover` and `defineReceiveCommand`.
I don't think it's consistent with `initalReceive`, but that might not matter. wdyt?
An alternative is to use `defineReceive` instead of `initialReceive`?